game/Board.ts

```diff
@@ -135,10 +140,55 @@ export class Board {
         return this.t[l.r][l.c];
     }
 
+    /*
+     * Getter for Tile and all of its neighbors at a location.
+     */
+    public getNeighborhood(l: Location): Tile[] {
+        let tiles: Tile[] = [];
+        for (let r = l.r - 1; r <= l.r + 1; r++) {
+            for (let c = l.c - 1; c <= l.c + 1; c++) {
+                if (this.inBounds(r, c)) {
+                    tiles.push(this.t[r][c]);
+                }
+            }
+        }
+        return tiles;
+    }
+
+    public getNearbyLocations(l: Location): Location[] {
```

Comment:
can these two "nearby location / tile" functions either call each other or use the map logic

Reply:
I forgot to remove getNeighborhood() sorry, it's never used. Only getNearbyLocations